hoot-core/src/main/cpp/hoot/core/io/OsmChangesetSqlFileWriter.cpp

```diff
@@ -170,13 +176,13 @@ ElementPtr OsmChangesetSqlFileWriter::_getChangeElement(ConstElementPtr element)
     default:
       throw HootException("Unknown element type");
   }
-
   return changeElement;
 }
 
 // If osm.changeset.file.writer.generate.new.ids is false, then these create methods assume
 // you've already set the ID correctly in terms of the OSM API target db for the element to be
 // created.
+//TODO: Can we get rid of the generate new ID's option now?
```

Comment:
So is it possible for derive changeset to not much with the db sequence of generate a non-sql changeset?  I agree that feature is still needed for a sql applied changeset.